app/src/main/java/app/revanced/manager/patcher/Session.kt

```diff
@@ -21,10 +25,15 @@ class Session(
     frameworkDir: String,
     aaptPath: String,
     multithreadingDexFileWriter: Boolean,
+    private val app: Context,
```

Comment:
Makes more sense to call it `context` because it is not an `Application`.

```suggestion
    private val context: Context,
```

Reply:
The name `context` is already used by `PatcherContext`

Reply:
That class is not referenced in this file

Reply:
it is in `Patcher.applyPatchesVerbose()`

Reply:
Thats `Patcher`, not `PatcherContext`